README.md

```diff
@@ -182,6 +187,18 @@ The internal storage of the products Charge Control C (Tarragon) and M (EVAcharg
 | /dev/mmcblk0p6 | 128 MB | Logging file system A (/var/log) |
 | /dev/mmcblk0p7 | 128 MB | Logging file system B (/var/log) |
 
+A similar partitioning is used on the Charge SOM based products:
```

Comment:
I think we should mention that this partitioning only apply to the eMMC.

Reply:
I now mentioned the eMMC